samples/grids/grid/cascading-combo/src/index.tsx

```diff
@@ -98,53 +98,47 @@ export default class Sample extends React.Component<any, any> {
         }
     }
 
-    public webGridWithComboRendered(gridRef: IgrGrid, args: IgrVoidEventArgs) {
-        gridRef.data = this.gridData;
+    public webGridWithComboRendered(args: IgrVoidEventArgs) {
+        args.target.data = this.gridData;
     }
 
-    public onCountryChange( rowId: string, cmp: any, args:any) {
+    public onCountryChange(rowId: string, args: CustomEvent<any>) {
         // find next combo
-        // args incomplete, so gte value from component on timeout as workaround.
         const regionCombo = this.comboRefCollection.get("region_" + rowId);
         const cityCombo = this.comboRefCollection.get("city_" + rowId);
         const regions = this.regions;
-       setTimeout(() => {
-            const newValue = cmp.value[0];
-            if (newValue === undefined) {
-                regionCombo.deselect(regionCombo.value);
-                regionCombo.disabled = true;
-                regionCombo.data = [];
-
-                cityCombo.deselect(regionCombo.value);
-                cityCombo.disabled = true;
-                cityCombo.data = [];
-            } else {
-                regionCombo.disabled = false;
-                regionCombo.data = regions.filter(x => x.Country === newValue);
-
-                cityCombo.deselect(cityCombo.value);
-                cityCombo.disabled = true;
-                cityCombo.data = [];
-            }
-       });
+        const newValue = args.detail.newValue[0];
+        if (newValue === undefined) {
+            regionCombo.deselect(regionCombo.value);
+            regionCombo.disabled = true;
+            regionCombo.data = [];
+
+            cityCombo.deselect(regionCombo.value);
+            cityCombo.disabled = true;
+            cityCombo.data = [];
+        } else {
+            regionCombo.disabled = false;
+            regionCombo.data = regions.filter(x => x.Country === newValue);
+
+            cityCombo.deselect(cityCombo.value);
+            cityCombo.disabled = true;
+            cityCombo.data = [];
+        }
     }
 
-    public onRegionChange( rowId: string, cmp: any, args:any) {
+    public onRegionChange(rowId: string, args: CustomEvent<any>) {
```

Comment:
@dkamburov Initially thought that didn't work, but saw imports in some templates, so.. these arg could/should be typed as `args: IgrComboChangeEventArgs` to have typing of detail and that seems to be true for almost all of the event handlers in the React samples.
I've restricted my changes to fixing them up for the update, but that doesn't mean they are 'good' now, just not broken and we should invest some time later to clean up all of this.